### PR TITLE
{chem}[foss/2018b,intel/2018b] ASE v3.19.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4d7998c09e13e9157943989502e9f4b4 for a full test report.

File: easybuild/easyconfigs/a/ASE/ASE-3.19.0-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    ('matplotlib', '3.0.0', '-Python-%(pyver)s'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# pytest-xdist does not install a separate module, but a plugin to pytest.
+# It therefore cannot be imported during the sanity check.
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/ASE/ASE-3.19.0-foss-2018b-Python-3.6.6.eb]: what does this comment refer to?

Comment (schiotz) [reply easybuild/easyconfigs/a/ASE/ASE-3.19.0-foss-2018b-Python-3.6.6.eb]: Sorry, that was a stale comment.  I removed pytest-xdist again, as it is only the developer version (coming 3.20.0) that uses it.  I forgot to remove the comment.

**Comment by migueldiascosta** [comment]
Going in, thanks @schiotz!

**Pull request merged**